packages/attribute-slicer/src/AttributeSlicer.ts

```diff
@@ -125,6 +125,23 @@ export class AttributeSlicer {
     private searchDebounce = SEARCH_DEBOUNCE;
 
     /**
+     * Whether or not to left align item text
+     */
+    private _leftAlignText = false;
+    get leftAlignText() {
+        return this._leftAlignText;
+    }
+    
+    /**
+     * Sets wheter or not to left align item text
+     */
+    set leftAlignText(value: boolean){
+        if (value !== this._leftAlignText){
+            this._leftAlignText = value;
```

Comment:
I would add a
```
if (this.virtualList) {
    this.virtualList.rerender();
}
```
I'm guessing that when you toggle the "Left Alignment" option in the Formatting Pane, that the slicer automagically updates?  If so, I'm pretty certain that is because AttributeSlicerVisual uses the `state` setter, and there are other things in there cause the list to rerender.  Think of the state setter as an optional way to set a bunch of configuration options at once.  For example, if AttributeSlicerVisual only set `slicer.leftAlignText = true`, then the list would not update correctly.